content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1344 @@
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```cmd
+<engine>/scripts/o3de.bat register --this-engine
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+For Linux, open a terminal. Then, run the O3DE Python runtime using one of the following ways, and enter the command you want to perform. Note: Replace `<engine>` with the path to your engine. 
+
+- Launch python and run the `o3de.py` script.
+
+    ```bash
+    <engine>/python/python.sh <engine/scripts/o3de.py
+    ```
+
+- Run the `o3de.sh` shell script file directly.
+
+    ```bash
+    <engine>/scripts/o3de.sh
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```bash
+<engine>/scripts/o3de.sh register --this-engine
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Commands
+
+
+| Command | Description | 
+| - | - |
+| [`get-global-project`](#get-global-project) | Gets the global project that is registered to the engine. |
+| [`set-global-project`](#set-global-project) | Sets the specified project as the engine's global project. |
+| [`create-template`](#create-template) | Creates a template out of the specified source path. |
+| [`create-from-template`](#create-from-template) | Creates a template from an existing template.  |
+| [`create-project`](#create-project) | Creates a new project. |
+| [`create-gem`](#create-gem) | Creates a new Gem. |
+| [`register`](#register) | Register an O3DE object. |
+| [`register-show`](#register-show) | Shows the registered O3DE objects. |
+| [`get-registered`](#get-registered) | Shows the registered O3DE object's path that is mapped to the specified name. |
+| [`enable-gem`](#enable-gem) | Enables the Gem in your project, so you can use the assets or code the Gem provides. |
+| [`disable-gem`](#disable-gem) | Disables the Gem in your project. |
+| [`sha256`](#sha256) | Creates a secure hash algorithm (SHA-256). |
+<!-- | [`edit-project-properties`](#edit-project-properties) |  | -->
+
+
+
+## `get-global-project`
+
+Gets the global project that is registered to the engine in the specified `.setreg` file at `<user>/.o3de/Registry/`.
+
+
+```cmd
+usage: o3de.py get-global-project [-h] [-i INPUT_PATH]
+```
+
+### Usage
+
+```cmd
+o3de.bat get-global-project -i SETREG_PATH
+```
+
+### Optional Parameters
+
+`-h, --help`
+show this help message and exit
+
+`-i INPUT_PATH, --input-path INPUT_PATH`
+
+Optional path to file to read `/Amazon/AzCore/Bootstrap/project_path` key from. If not supplied, then
+`C:\Users\USER_PATH\.o3de\Registry\bootstrap.setreg` is used instead
+
+
+
+## `set-global-project`
+
+Sets the specified project as the engine's global project in the specified `.setreg` file at `<user>/.o3de/Registry/`
+
+```cmd
+usage: o3de.oy set-global-project [-h] (-pp PROJECT_PATH | -pn PROJECT_NAME)
+                                  [-o OUTPUT_PATH] [-f]
+```
+
+### Usage
+
+```cmd
+o3de.bat set-global-project -pp PROJECT_PATH -o SETREG_PATH
+```
```

Comment:
Refer to the get-global-project command for suggestions on the formatting of this standard info.